base_wamas/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Base WAMAS",
+    "summary": """Base module to aggregate WAMAS features.""",
+    "version": "16.0.1.0.0",
+    "development_status": "Beta",
+    "website": "https://github.com/OCA/edi",
+    "license": "LGPL-3",
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "maintainers": ["tuantrantg"],
```

Comment:
this is a strong commitment my friend

Reply:
hi @nilshamerlinck the PR's updated, please review it again, thank you